src/tools/rustbook/Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
-authors = ["The Rust Project Developers"]
+authors = ["Steve Klabnik <steve@steveklabnik.com>"]
```

Comment:
Could this stay as "The Rust Project Developers"? I think we just try to have that as an author for all the crates in here.